app/src/main/res/layout/dialog_qr_code_generator.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <ScrollView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:orientation="vertical"
+                android:gravity="center">
+
+                <RelativeLayout
+                    android:id="@+id/relativeLayout"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="4dp"
+                    android:layout_marginTop="30dp"
+                    android:layout_marginEnd="4dp"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/linearLayout">
+
+                    <EditText
```

Comment:
what is the purpose of this input?

Reply:
Hey @druchniewicz . Thanks for this, My initial design was to be able to see generated QR code with 8 digits, When a user clicks ok, then this same unique id should appear under child Id.

However one thing, this child Id reflects but is not being saved while registration. so probably something is not being mapped.

Reply:
Here the updated ui  ideally if a user clicks on generate QR code, the generated code should be displayed on child id which is currently working but its not saved.
![Screenshot from 2024-11-12 18-25-50](https://github.com/user-attachments/assets/756b8068-2629-48ee-9d71-fb8655db1b71)